client/src/ui/organisms/article-actions/language-menu/index.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-no-target-blank */
```

Comment:
Why not add the required `rel="noreferrer"` to the link below? https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md

Reply:
We don't set the `rel=""` attribute on external links on content, and the default `Referrer-Policy` ([strict-origin-when-cross-origin](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy#strict-origin-when-cross-origin)) applies here, so GitHub will only see `https://developer.mozilla.org`, for which I see no reason to hide it.

Reply:
@LeoMcA Instead of the `eslint-disable`, I now reconfigured the rule in https://github.com/mdn/yari/pull/11865/commits/1fd919f307546ceb63d8a47de76d6ea8fb7b0732. Let me know what you think.

Reply:
(Let me move this into another PR.)

Reply:
See: https://github.com/mdn/yari/pull/11867